lib/cisco_node_utils/bgp_af.rb

```diff
@@ -500,8 +491,7 @@ def default_maximum_paths_ibgp
 
     # Build an array of all network commands currently on the device
     def networks
-      cmds = config_get('bgp_af', 'network', @get_args)
-      cmds.nil? ? default_networks : cmds.each(&:compact!)
+      config_get('bgp_af', 'network', @get_args).each(&:compact!)
```

Comment:
>You're mostly correct but in this case compact returns nil within the scope of the iterator; the end result is that cmds is still returned as an array, changed or otherwise.

Ack